src/hotspot/share/gc/g1/g1_globals.hpp

```diff
@@ -73,7 +73,11 @@
                                                                             \
   product(bool, G1EvacuationFailureALotDuringMixedGC, true,                 \
           "Force use of evacuation failure handling during mixed "          \
-          "evacuation pauses")
+          "evacuation pauses")                                              \
+                                                                            \
+  product(uintx, G1EvacuationFailureALotCSetPercent, 100,                   \
+          "Percent of the regions in cset that will fail evacuation")       \
```

Comment:
Please spell out "cset" in the description text though as it is done everywhere else in `g1_globals.hpp`. Considering Albert's suggestion, maybe something like:

```
The percentage of regions in the collection set starting from the beginning where the forced evacuation failure injection will be applied.
```

Reply:
Sure. Thanks!